schema/Query.graphql

```diff
@@ -28,4 +28,13 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Get life
+    """
+    getLife(id: ObjectID!): Life!
```

Comment:
The return here should be optional. Since it is possible that the id passed here may not exist